lookout/style/format/feature_extractor.py

```diff
@@ -185,21 +160,26 @@ def populate_indices(feature_group: FeatureGroup, feature_names: Sequence[str],
         # we don't need the last `total_index`
 
         self._feature_node_counts = {group: [sum(len(feature) for feature in node_index.values())
-                                             for node_index in self.feature_to_indices[group]]
+                                             for node_index in self._feature_to_indices[group]]
                                      for group in FeatureGroup}
         self._feature_group_counts = {group: sum(counts)
                                       for group, counts in self._feature_node_counts.items()}
         self._feature_count = sum(self._feature_group_counts.values())
 
     @property
-    def index_to_feature(self) -> IndexToFeature:
+    def index_to_feature(self) -> Optional[IndexToFeature]:
         """Return the mapping from integer indices to the corresponding feature names."""
-        return self._index_to_feature
+        return self._index_to_feature if hasattr(self, "_index_to_feature") else None
```

Comment:
I think it is better to raise an error in case there is no `_index_to_feature`. It is kind of unexpected situation.